Main/Source/iconf.cpp

```diff
@@ -669,6 +642,9 @@ void ivanconfig::ShowItemsAtPlayerSquareChanger(cycleoption* O, long What)
 {
 //  if(What==1 && GetStartingDungeonGfxScale()<3){ //above head asked
 //    What=2; //upgrade to 1st corner
+//  }
+//  if(What==1 && GetStartingFontGfx()<3){ //I have no idea what these lines mean, copying font stuff in case it's ever important --Stihdjia
```

Comment:
this block of comments/deadCode can be safely removed (is just some stuff I left behind and forgot to clean), 
the automatic corners calculation is elsewhere now